test/inductor/test_torchinductor.py

```diff
@@ -5058,6 +5058,25 @@ def fn(x):
         )
 
 
+def _add_dynamic_shapes_test(test_name: str):
+    make_test_cls_with_patches(
+        cls=CommonTemplate,
+        make_new_cls=False,
+        new_fn_suffix="_dynamic_shapes",
+        name_pred=lambda name: name.startswith(test_name),
+        patches=(
+            (config, "dynamic_shapes", True),
+            (torch._dynamo.config, "dynamic_shapes", True),
+            (functorch_config, "use_dynamic_shapes", True),
+        ),
+    )
```

Comment:
It feels better to do the same pattern as test_dynamic_shapes.py in torchdynamo, because in the end state, we want ALL of the tests passing, and having to manually reenumerate every test here to enable it testing on dynamic shapes is bad (because people will certainly forget). You have to do a big bang "disable most tests as not working on dynamic shapes" to start with (write a script to generate this list for you), but it puts us in a better spot for forward looking dev. 

cc @ngimel @Chillee for other opinions